src/Console/InstallCommand.php

```diff
@@ -513,7 +513,7 @@ protected function installInertiaSsrStack()
         });
 
         copy(__DIR__.'/../../stubs/inertia/resources/js/ssr.js', resource_path('js/ssr.js'));
-        $this->replaceInFile("input: 'resources/js/app.js',", "input: 'resources/js/app.js',".PHP_EOL."            ssr: 'resources/js/ssr.js',", base_path('vite.config.js'));
+        copy(__DIR__.'/../../stubs/inertia/vite.config.ssr.js', base_path('vite.config.js'));
```

Comment:
@timacdonald What's the reason for creating a dedicated SSR config? Can't we just add the `ssr.noExternal` stuff to the main config if they've installed with `--ssr`?